waddrmgr/manager_test.go

```diff
@@ -1148,7 +1169,9 @@ func testChangePassphrase(tc *testContext) bool {
 	}
 
 	// Ensure the public passphrase was successfully changed.
-	if !tc.rootManager.TstCheckPublicPassphrase(pubPassphrase2) {
+	secretKey := snacl.SecretKey{Key: &snacl.CryptoKey{}}
```

Comment:
Possible to get the secret key from the root manager here? 

Hard to follow what is going on here, at least add a comment if not possible.

Reply:
Added a comment. IIUC, the point of this snippet is to ensure that we can re-derive the public key with the new passphrase.